pkg/ui/ccl/src/views/shared/containers/licenseSwap/index.tsx

```diff
@@ -53,11 +53,11 @@ export default function swapByLicense<TProps>(
   OSSComponent: React.ComponentClass<TProps>,
   CCLComponent: React.ComponentClass<TProps>,
   // tslint:enable:variable-name
-): React.ComponentClass<TProps> {
+) {
   const ossName = getComponentName(OSSComponent);
   const cclName = getComponentName(CCLComponent);
 
-  class LicenseSwap extends React.Component<TProps & OwnProps> {
+  class LicenseSwap extends React.Component<TProps & OwnProps & any> {
```

Comment:
Following hack with **&any** type is required to make changes related to types only and do not rewrite component logic.